### PR TITLE
Add Divide<> function

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/divide.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#ifndef FUNCTIONS_DIVIDE_H
+#define FUNCTIONS_DIVIDE_H
+
+// Usage: Divide<F, V>
+// Divide F by V
+// If V = 0, returns 0
+// F, V: INTEGER,
```
Comment (profezzorn) [inline functions/divide.h:7]: INTEGER -> FUNCTION

File: functions/divide.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    v_.run(blade);
+  }
+  int getInteger(int led) {
+    if (v_ == 0) {
+      return 0;
+    } else {
+    return (f_.getInteger(led) / v_.getInteger(led));
```
Comment (profezzorn) [inline functions/divide.h:23]: not indented correctly.
Remove extra parenthesis.

File: functions/divide.h [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    v_.run(blade);
+  }
+  int calculate(BladeBase* blade) {
+    if (v_ == 0) {
+      return 0;
+    } else {
+      return (f_.calculate(blade) / v_.calculate(blade));
```
Comment (profezzorn) [inline functions/divide.h:42]: extra parenthesis

File: props/saber_fett263_buttons.h [hunk 4]
```diff
@@ -34,6 +34,7 @@ Standard Controls While Blade is OFF
     Turn Left (Stepped) = Previous Preset
       Increment by 5 = Hold PWR + Turn Left
     Click PWR = Select Preset
+      * if FETT263_QUICK_SELECT defined, blade will ignite with preset selection
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:37]: Doesn't belong in this PR (same with lots of things below)

**Comment by fdarosa2663** [comment]
wrong branch, will submit new branch PR
